sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs

```diff
@@ -675,6 +675,47 @@ public async Task CanQueryWithFormattedQuery(FormattableStringWrapper query)
             Assert.True(response.Value.Single());
         }
 
+        [RecordedTest]
+        public async Task CanQueryResource()
+        {
+            var client = CreateClient();
+
+            var results = await client.QueryResourceAsync(TestEnvironment.StorageAccountId,
+                "search *",
+                _logsTestData.DataTimeRange);
+
+            Assert.AreEqual(LogsQueryResultStatus.Success, results.Value.Status);
+            var resultTable = results.Value.Table;
+            CollectionAssert.IsNotEmpty(resultTable.Rows);
+            CollectionAssert.IsNotEmpty(resultTable.Columns);
+
+            bool verifyRow = false;
+            bool verifyColumn1 = false;
+            bool verifyColumn2 = false;
+            foreach (LogsTableRow rows in resultTable.Rows)
+            {
+                foreach (var row in rows)
+                {
+                    if ((row != null) && row.ToString().Contains("Create/Update Storage Account"))
```

Comment:
Is the intention here is to check whether each row has this value or it is at least true once? The current logic only tests for it once.

Reply:
Actually - this one is still a question. Do you want to check if each row has this string or is the intend to check if one of the rows has this string?